client/src/styles/globals.css

```
@@ -100,6 +110,46 @@ html {
+
+.fullscreen-button:hover {
+  transform: scale(1.1);
+}
+
+/* Mobile full screen optimizations */
+@media screen and (max-width: 768px) {
+  html,
+  body {
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
+    -khtml-user-select: none;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)